sideload-repos-systemd/flatpak-create-sideload-symlinks.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# This script is intended to be run by flatpak-sideload-usb-repo.service
+
+if ! test $# -eq 1 || ! test -d "$1"; then
+  echo "Error: first argument must be a directory"
+  exit 1
+fi
+
+# Add a link to any newly inserted drives which were copied to with "flatpak create-usb"
+for f in $1/*; do
+    if ! test -d "$f"; then
+        continue
+    fi
+    base=$(basename "$f")
+    if test -e "/run/flatpak/sideload-repos/$base"; then
+        continue
+    fi
+    if test -d "$f/ostree/repo" || test -d "$f/.ostree/repo" || test -d "$f/.ostree/repos.d"; then
+        ln -s "$f" "/run/flatpak/sideload-repos/$base";
+    fi
+done
+
+# Remove any broken symlinks e.g. from drives that were removed
+for f in /run/flatpak/sideload-repos/*; do
```

Comment:
Also, should maybe have a prefix so we don't remove unrelated links here?

Reply:
I was thinking a broken link probably shouldn't be there anyway, but you're right it would be cleaner to only remove our own stuff. I can use `automount-` as a prefix I suppose.